saleor/graphql/order/mutations/orders.py

```diff
@@ -420,33 +421,68 @@ def perform_mutation(cls, _root, info, amount, **data):
         return OrderRefund(order=order)
 
 
-class OrderUpdateMeta(UpdateMetaBaseMutation):
+class OrderUpdateMetaBaseMutation(UpdateMetaBaseMutation):
+    class Meta:
+        abstract = True
+
+    class Arguments:
+        token = graphene.UUID(
+            description="Token of an object to update.", required=True
+        )
+        input = MetaInput(
+            description="Fields required to update new or stored metadata item.",
+            required=True,
+        )
+
+    @classmethod
+    def get_instance(cls, info, **data):
+        token = data["token"]
+        return models.Order.objects.get(token=token)
+
+
+class OrderClearMetaBaseMutation(ClearMetaBaseMutation):
+    class Meta:
+        abstract = True
+
+    class Arguments:
+        token = graphene.UUID(description="Token of an object to clear.", required=True)
+        input = MetaPath(
+            description="Fields required to update new or stored metadata item.",
+            required=True,
+        )
+
+    @classmethod
+    def get_instance(cls, info, **data):
+        token = data["token"]
```

Comment:
@maarcingebala public meta will use `token` as ID. Should we do the same for private meta? Private is protected by `order.manage_orders` so we could also use a normal ID here.

Reply:
Aaaand done ;)